modules/smart-agent_redis/conf/10-hitrate.yaml

```diff
@@ -14,12 +14,12 @@ signals:
     formula: (A/(A+B)).scale(100)
 
 rules:
-  critical:
```

Comment:
Please keep `critical` as disabled by default to avoid making a major release with breaking changes.